Dockerfile

```diff
@@ -1,15 +1,17 @@
-FROM python:3.9
+FROM python:3.11
 LABEL maintainer="llindstrom@lco.global"
 
 # the exposed port must match the deployment.yaml containerPort value
 EXPOSE 80
-ENTRYPOINT [ "/usr/local/bin/gunicorn", "tom_demo_base.wsgi", "-b", "0.0.0.0:80", "--access-logfile", "-", "--error-logfile", "-", "-k", "gevent", "--timeout", "300", "--workers", "2"]
+ENTRYPOINT [ "poetry", "run", "gunicorn", "tom_demo_base.wsgi", "-b", "0.0.0.0:80", "--access-logfile", "-", "--error-logfile", "-", "-k", "gevent", "--timeout", "300", "--workers", "2"]
 
 WORKDIR /tom-demo
 
+RUN pip install --upgrade pip && pip install 'poetry >=2.0,<3.0'
+
 COPY . /tom-demo
-RUN pip install --upgrade pip && pip install poetry
 RUN poetry config virtualenvs.create false --local
+RUN poetry lock
```

Comment:
Do we want to re-generate the poetry.lock file here?  I thought the point of having the lock file in the repo was that it specifies the intended set of dependencies, and this would generate them all over again and they might be now different than the ones intended. Or, am I thinking about that incorrectly?

Reply:
Fixed.